Samples/RenderGraphViewer/RenderGraphViewer.cpp

```diff
@@ -134,31 +208,42 @@ void RenderGraphViewer::loadScene(const std::string& filename, bool showProgress
     mpScene->getActiveCamera()->setAspectRatio((float)pSample->getCurrentFbo()->getWidth() / (float)pSample->getCurrentFbo()->getHeight());
 }
 
+void RenderGraphViewer::fileWriteCallback(const std::string& fileName)
+{
+    std::ifstream inputStream(fileName);
```

Comment:
Any guarantee that this function runs on the same thread as the renderer?
If not, need to make sure we don't have a race condition